assets/dev/js/editor/elements/views/base.js

```diff
@@ -1106,6 +1147,29 @@ BaseElementView = BaseContainer.extend( {
 			groups: [ 'elementor-element' ],
 		} );
 	},
+
+	getRepeaterItemActiveIndex() {
+		return this.getContainer().renderer.view.model.changed.editSettings.changed.activeItemIndex;
+	},
+
+	tryHandleDynamicTagsAdvancedTools( dataBinding, settings ) {
+		const attributeKeys = Object.keys( settings.attributes ),
+			shouldTryHandle = ! this.findUniqueKey( [ 'before', 'after', 'fallback' ], attributeKeys, true ).length,
+			changedControlKey = Object.keys( settings.changed )[ 0 ];
+
+		if ( ! shouldTryHandle || ! changedControlKey ) {
+			return false;
+		}
+
+		jQuery( dataBinding.el ).html( this.getAdvancedDynamicTitleChange(
```

Comment:
Tried using `dataBinding.el.innerHTML = ...` but it didn't render changes until entire process was done, caused delays and failed tests